src/renderer/components/ft-input/ft-input.js

```diff
@@ -35,6 +35,10 @@ export default Vue.extend({
       type: Boolean,
       default: false
     },
+    spellcheck: {
+      type: String,
+      default: ''
+    },
```

Comment:
Make this a default `true` `Boolean` instead, `String` makes it look like it can have more than 2 states.
```suggestion
    spellcheck: {
      type: Boolean,
      default: true
    },
```